cloudconfig/instancecfg/instancecfg.go

```diff
@@ -558,6 +572,27 @@ func copyToolsList(in coretools.List) coretools.List {
 	return out
 }
 
+func (cfg *InstanceConfig) SetSnapSource(snapPath string, snapAssertionsPath string) error {
+	if snapPath == "" {
+		return nil
+	}
+
+	_, err := os.Stat(snapPath)
+	if err != nil {
+		return errors.Annotatef(err, "unable to find snap at %s", snapPath)
```

Comment:
The underlying error message from os.Stat() will be similar to the annotate string so the final message will be a bit jittery. Perhaps reword a bit?